### PR TITLE
Fix and update attribute-behavior fixture

### PR DESCRIPTION
**Pull request by eps1lon**
## Summary

Due to https://github.com/facebook/react/issues/25928 the attribute fixture could no longer finish since it expects at least something to render. But since Fizz currently breaks down completely on malformed `<meta>` tags, the fixture could no longer handle this.

The fixture now renders valid types for `meta` tags.

Note that the snapshot change to `viewTarget`` is already on `main`. Review by commit helps to understand this.

Added `html[lang]` so that we test at least one standard attribute on `<html>`. `version` is obsolete so results are not that trustworthy.

## How did you test this change?

With Chrome Version 109.0.5414.119 (Official Build) (64-bit)

- `yarn build --type=UMD_DEV react/index,react-dom && cd fixtures/attribute-behavior && yarn install && yarn start`

### CODE REVIEW

File: fixtures/attribute-behavior/AttributeTableSnapshot.md
Comment (eps1lon) [inline fixtures/attribute-behavior/AttributeTableSnapshot.md:12204]: Caused by https://github.com/facebook/react/issues/26128

**Comment by gnoff** [comment]
@eps1lon once #26106 lands the meta tags will get hoisted but won't require the extra props to avoid getting dropped. do you want to wait for that to land this or undo these changes once that one is incorporated? Also is this running on oss-stable only? It seems it is not flagged but we could in theory have singletons and hoistables on for some builds and not others

**Comment by eps1lon** [comment]
> @eps1lon once https://github.com/facebook/react/pull/26106 lands the meta tags will get hoisted but won't require the extra props to avoid getting dropped. do you want to wait for that to land this or undo these changes once that one is incorporated?

I think it's better to wait for #26106 for now. 

> Also is this running on oss-stable only?

It's running on the published latest and on `main`. But I think only `main` is actually used for the snapshot.

File: fixtures/attribute-behavior/src/App.js
Comment (eps1lon) [inline fixtures/attribute-behavior/src/App.js:287]: Consistent with how we read the value of server-rendered markup. `firstChild` returns the DOCTYPE now if we render into a `document` on the client.

File: fixtures/attribute-behavior/src/attributes.js
Comment (gnoff) [inline fixtures/attribute-behavior/src/attributes.js:363]: Do you want to remove the extra props to assert the tag still shows up with it has minimal props?

File: fixtures/attribute-behavior/src/App.js
Comment (gnoff) [inline fixtures/attribute-behavior/src/App.js:775]: What caused this change?

File: fixtures/attribute-behavior/src/attributes.js
Comment (eps1lon) [reply fixtures/attribute-behavior/src/attributes.js:363]: Definitely if it works now.

File: fixtures/attribute-behavior/src/App.js
Comment (eps1lon) [reply fixtures/attribute-behavior/src/App.js:775]: `renderToString` is no longer included in `react-dom-server.browser`. Makes sense considering `renderToString` is considered legacy. `renderToString` was already excluded from the [18.0.0 UMD bundle](https://unpkg.com/react-dom@18.0.0/umd/react-dom-server.browser.development.js) so there was no change to the UMD bundle in a SemVer minor.

File: fixtures/attribute-behavior/src/attributes.js
Comment (eps1lon) [reply fixtures/attribute-behavior/src/attributes.js]: Removed the `extraProps` since the fixture works without them. Thank you for pointing this out!

File: fixtures/attribute-behavior/src/App.js
Comment (gnoff) [reply fixtures/attribute-behavior/src/App.js:775]: oh odd, I thought that was already the case. sounds good

**Review by gnoff — APPROVED** [review]

**Pull request merged**
